src/styles.css

```diff
@@ -234,4 +289,74 @@ button {
 .pmndrs-menu a:hover {
   text-decoration: underline;
   color: inherit;
-}
\ No newline at end of file
+}
+
+@media (max-width: 768px) and (max-aspect-ratio: 13/9) {
+  .controls-container {
+    gap: 0;
+  }
+}
+
+@media (max-width: 960px) and (min-aspect-ratio: 13/9) {
+  .stack {
+    align-items: center;
+    padding-bottom: 4rem;
+  }
+  .keys {
+    grid-template-columns: 1fr 1fr;
+    column-gap: 1rem;
+  }
+
+  .mobile-controls {
+    padding: 0 1rem;
+  }
+
+  .controls {
+    bottom: initial;
+    top: 30px;
+    right: 30px;
+  }
+  .speed {
+    bottom: initial;
+    top: 40px;
+    left: 30px;
+  }
+}
+
+@media (max-width: 767px) {
+  .pmndrs-menu {
+    flex-wrap: wrap;
+    gap: 0.5rem 1.25rem;
+  }
+  .pmndrs-menu > div {
+    padding: 0;
+  }
+}
+
+@media (hover: hover) {
```

Comment:
Hmm we are now going to rely into media queries to detect, touch devices? Well I have no experience with this specific media query and don't know the reliability of this. It should be hold in mind that it could break with edge cases.

Reply:
Thereby a detect if any keys are pressed(mobiles/tablets with keyboards) that we could add a `non-touch` class into the html and then as well hide these.

Reply:
I suggest we use a package to handle this: https://github.com/duskload/react-device-detect

so we don't have to worry about the edge cases